src/test/java/com/minecolonies/coremod/entity/ai/util/ChatSpamFilterTest.java

```diff
@@ -29,79 +30,79 @@ public void setup()
     @Test
     public void testFilterFirst()
     {
-        filter.requestWithoutSpam(MESSAGE_1);
+        filter.requestWithoutSpam(new TextComponentString(MESSAGE_1));
```

Comment:
Can't we just make the chat spam filter make this TextComponentString if we provide a string?